recipe/meta.yaml

```diff
@@ -39,6 +41,7 @@ test:
         - fiona.fio
     commands:
         - fio --help
+        - conda inspect linkages fiona --name _test  # [linux]
```

Comment:
Is there an assertion we can make here? The issue itself was with osx - I'd really love to see a test for that here... eventually we are going to tool ourselves up so that this kind of thing is trivial (i.e. ```assert_links $SP_DIR/fiona/ogrext* libgif* $PREFIX```